packages/skeleton/src/lib/components/Avatar/Avatar.svelte

```diff
@@ -53,18 +53,17 @@
 
 <!-- FIXME: resolve a11y warnings -->
 <!-- svelte-ignore a11y-no-noninteractive-element-interactions -->
-<figure class="avatar {classesBase}" data-testid="avatar" on:click on:keydown on:keyup on:keypress>
-	{#if src}
+<figure class="avatar {classesBase}" data-testid="avatar" {...prunedRestProps()} on:click on:keydown on:keyup on:keypress>
```

Comment:
be aware that moving the `restProps` from the image to the figure might break behavior that was previously established. Although I'm not sure how much of a change this might be and how the `figure` and `img` props actually differ.

Reply:
I had a similar concern, but this was an undocumented feature, so usage should be fairly low. It wasn't very practical being applied to the image itself anyways.

Reply:
I agree, just wanted to put it on your mind :)